openmetadata-service/src/main/java/org/openmetadata/service/secrets/converter/ClassConverterFactory.java

```diff
@@ -46,25 +46,26 @@ private ClassConverterFactory() {
             Map.entry(BigQueryConnection.class, new BigQueryConnectionClassConverter()),
             Map.entry(BigTableConnection.class, new BigTableConnectionClassConverter()),
             Map.entry(DatalakeConnection.class, new DatalakeConnectionClassConverter()),
-            Map.entry(MysqlConnection.class, new MysqlConnectionClassConverter()),
-            Map.entry(HiveConnection.class, new HiveConnectionClassConverter()),
-            Map.entry(TrinoConnection.class, new TrinoConnectionClassConverter()),
-            Map.entry(PostgresConnection.class, new PostgresConnectionClassConverter()),
             Map.entry(DbtGCSConfig.class, new DbtGCSConfigClassConverter()),
             Map.entry(DbtPipeline.class, new DbtPipelineClassConverter()),
+            Map.entry(ElasticSearchConnection.class, new ElasticSearchConnectionClassConverter()),
             Map.entry(GCSConfig.class, new GCPConfigClassConverter()),
             Map.entry(GCPCredentials.class, new GcpCredentialsClassConverter()),
             Map.entry(GCSConnection.class, new GcpConnectionClassConverter()),
-            Map.entry(ElasticSearchConnection.class, new ElasticSearchConnectionClassConverter()),
+            Map.entry(HiveConnection.class, new HiveConnectionClassConverter()),
+            Map.entry(IcebergConnection.class, new IcebergConnectionClassConverter()),
+            Map.entry(IcebergFileSystem.class, new IcebergFileSystemClassConverter()),
             Map.entry(LookerConnection.class, new LookerConnectionClassConverter()),
-            Map.entry(SSOAuthMechanism.class, new SSOAuthMechanismClassConverter()),
+            Map.entry(MysqlConnection.class, new MysqlConnectionClassConverter()),
+            Map.entry(PostgresConnection.class, new PostgresConnectionClassConverter()),
+            Map.entry(SapHanaConnection.class, new SapHanaConnectionClassConverter()),
```

Comment:
Just added `SapHanaConnection` as new and sorted the list alphabetically for easier scan